src/utils/rest-api.js

```diff
@@ -389,3 +399,30 @@ export function fetchAppsAndUrls() {
         });
     });
 }
+
+export function requestNetworkChange(
+    userId,
+    studyName,
+    equipmentType,
+    equipmentId,
+    changeRequest
+) {
+    console.info('request network change');
+    const changeUrl =
+        getStudyUrl(studyName, userId) +
+        '/network-modification/' +
+        equipmentType +
+        '/' +
+        equipmentId;
+    console.debug(changeUrl);
+    return backendFetch(changeUrl, {
+        method: 'POST',
```

Comment:
POST or PUT ? (see remarks in gridsuite/network-modification-server#10)

Reply:
post, it has a body